lib/block_view_message.go

```diff
@@ -825,6 +849,71 @@ func (bav *UtxoView) _connectMessagingGroup(
 		}
 	}
 
+	// V3 Messages: MUTING and UNMUTING members
+	if blockHeight >= bav.Params.ForkHeights.DeSoV3MessagesBlockHeight {
+		if existingEntry != nil && !existingEntry.isDeleted {
+			value, operationTypeExists := txn.ExtraData["OperationType"]
+			if operationTypeExists {
+				// inline helper method
+				contains := func(memberList []*MessagingGroupMember, member *MessagingGroupMember) bool {
+					for _, a := range memberList {
+						if a == member {
+							return true
+						}
+					}
+					return false
+				}
+
+				if string(value) == "MessagingGroupOperationMute" {
```

Comment:
nit: can we make `MessagingGroupOperationMute` a constant? Same with unmute.

Reply:
done